homework_4/01_bets_api/cmd/bootstrap/api.go

```diff
@@ -0,0 +1,25 @@
+package bootstrap
+
+import (
+	"github.com/superbet-group/code-cadets-2021/homework_4/01_bets_api/cmd/config"
+	"github.com/superbet-group/code-cadets-2021/homework_4/01_bets_api/internal/api"
+	"github.com/superbet-group/code-cadets-2021/homework_4/01_bets_api/internal/api/controllers"
+	"github.com/superbet-group/code-cadets-2021/homework_4/01_bets_api/internal/domain/services"
+	"github.com/superbet-group/code-cadets-2021/homework_4/01_bets_api/internal/infrastructure/sqlite"
+)
+
+func newBetService(dbExecutor sqlite.DatabaseExecutor) *services.BetService {
+	return services.NewBetService(dbExecutor)
+}
+
+func newController(eventService controllers.DbService, dbExecutor sqlite.DatabaseExecutor) *controllers.Controller {
+	return controllers.NewController(eventService, dbExecutor)
+}
+
+// Api bootstraps the http server.
+func Api(dbExecutor sqlite.DatabaseExecutor) *api.WebServer {
+	eventService := newBetService(dbExecutor)
```

Comment:
you are missing a repository here.
Controller should have Service as a dependency and Service should have Repository as a dependency:
`Controller -> Service -> Repository`